docs/en/04_Changelogs/4.9.0.md

```diff
@@ -12,22 +12,35 @@
 
 ### Image lazy loading {#image-lazy-loading}
 
-Silverstripe CMS now lazy loads most images by default and includes some options to opt-out of this behaviour.
+Most modern browsers support the ability to "lazy load" images. When an image is configured to be 
+lazy loaded, browsers will only request the image once it's about to be visible to the users. This 
+reduces the initial rendering time for pages. 
 
-Image lazy loading defers the loading of images not in the viewport to improve the initial page load performance. 
-Read more about (image lazy-loading)[https://web.dev/browser-level-image-lazy-loading/].
+From v4.9, Silverstripe CMS lazy loads most images by default. You have the options to opt-out of this 
```

Comment:
```suggestion
From v4.9, Silverstripe CMS will lazy load most images by default. You have options available to opt-out of this 
```

Reply:
Changelogs should be intemporal ... that's why I think we should avoid using the future tense and default to present tense. I've updated the rest of the changelog accordingly.